nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -1317,6 +1318,82 @@ cluster if restarted and the flow for the cluster has not changed. If the flow w
 the removed node should be deleted before restarting the node to allow it to obtain the cluster flow (otherwise
 an uninheritable flow file exception may occur).
 
+=== File Manager
+
+The File Manager utility allows system administrators to take a backup of an existing NiFi installation, install a new version of NiFi
+in a designated location (while migrating any previous configuration settings) or restore an installation from a previous backup.
+File Manager supports NiFi version 1.0.0 and higher and is available in 'file-manager.bat' file for use on Windows machines.
+
+To show help:
+
+  file-manager.sh -h
+
+The following are available options:
+
+* `-o,--operation <arg>` File operation (install | backup | restore)
+* `-b,--backupDir <arg>` Backup NiFi Directory (used with backup or restore operation)
+* `-c,--nifiCurrentDir <arg>` Current NiFi Installation Directory (used optionally with install or restore operation)
+* `-d,--nifiInstallDir <arg>` NiFi Installation Directory (used with install or restore operation)
+* `-i,--installFile <arg>` NiFi Install File (used with install operation)
+* `-r,--nifiRollbackDir <arg>` NiFi Installation Directory (used with install or restore operation)
+* `-t,--bootstrapConf <arg>` Current NiFi Bootstrap Configuration File (used optionally)
+* `-m,--moveRepositories` Allow repositories to be moved to new/restored nifi directory from existing installation, if available (used optionally with install or restore operation)
+* `-x,--overwriteConfigs` Overwrite existing configuration directory with upgrade changes (used optionally with install or restore operation)
+* `-v,--verbose` Verbose messaging (optional)
+* `-h,--help` Print help info (optional)
+
+Example usage on Linux:
+
+ # backup NiFi installation
+ # option -t may be provided to ensure backup of external boostrap.conf file
+ ./file-manager.sh
+ -o backup
+ –b /tmp/nifi_bak
+ –c /usr/nifi_old
+ -v
+
+ # install NiFi using compressed tar file into /usr/nifi directory (should install as /usr/nifi/nifi-1.3.0).
```

Comment:
We should notify the user that if they upgrade with -m to also move repositories, start nifi, then try to downgrade, it's likely that the older implementation will not be able to read the newer repos.